README.md

```diff
@@ -1,225 +1,375 @@
+- [Workspace](#workspace)
+  - [Applications](#applications)
+  - [Libraries](#libraries)
+  - [Tools](#tools)
+- [Scripts](#scripts)
+- [Workflows (CI/CD)](#workflows-cicd)
+- [Visual Studio Code](#visual-studio-code)
+
+## General
+
+> The documentation in this README assumes the reader has a general understanding
+> of [Nx] and [Angular]. For beginners in these technologies, the
+> [Core Nx Tutorial](https://nx.dev/getting-started/core-tutorial) and
+> [Angular Tour of Heroes](https://angular.io/tutorial) serves as a good
+> introduction.
+
+This repository is a monorepo which hosts serveral applications that all share
+the same dependencies (for example, every application is running the same
+version of Angular).
+
+_Note: Since this is an [Nx](https://nx.dev) workspace, the Nx CLI should be used
+over the Angular CLI._
+
+## Prerequisites
+
+<dl>
+  <dt><a href="https://volta.sh">Volta</a></dt>
+  <dd>Manager for JavaScript command-line tools like Node.js® and Yarn.</dd>
+  <dt><a href="https://dotnet.microsoft.com/en-us/download">.NET SDK</a></dt>
+  <dd>Required for running and developing the Backend For Frontend.</dd>
+</dl>
+
+## DataHub
+
+The application is deployed to five different environments as listed below:
+
+| Development       | Development\*     | Test              | Pre-production    | Production        |
+| ----------------- | ----------------- | ----------------- | ----------------- | ----------------- |
+| [U-001][dh-u-001] | [U-002][dh-u-002] | [T-001][dh-t-001] | [B-001][dh-b-001] | [B-002][dh-b-002] |
+
+[dh-u-001]: https://jolly-sand-03f839703.azurestaticapps.net
+[dh-u-002]: https://wonderful-field-057109603.1.azurestaticapps.net
+[dh-t-001]: https://lively-river-0f22ad403.azurestaticapps.net
+[dh-b-001]: https://blue-rock-05b7e5e03.azurestaticapps.net
+[dh-b-002]: https://purple-forest-07e41fb03.azurestaticapps.net
+
+<sub>\* This is identical to **U-001**, except it also hosts
+[B2C](https://azure.microsoft.com/en-us/services/active-directory/external-identities/b2c/).
+This service can be accessed from **localhost**, **U-001** and **U-002**.</sub>
+
+### Documentation
+
+Be sure to check out the additional DataHub documentation for the following
+areas:
+
+- [Testing](docs/dh/testing.md)
+- [Mocking](docs/dh/mocking.md)
+- [Feature Flags](libs/dh/shared/feature-flags/README.md)
+- [Logging with Application Insights](docs/dh/logging.md)
+
+### Getting Started
+
+Use the following command to serve the DataHub application locally:
+
+```sh
+yarn nx serve app-dh
 ```
 
-### Folder Structure - library
+The application utilizes request mocking for some of the requests to the
+[backend for frontend (BFF)](#backend-for-frontend-bff), but there are still
+features that are not mocked. When working with those features, it might be
+required to serve the BFF locally as well. To do so, run the following command
+(requires some initial setup, see
+[Setup of BFF](apps/dh/api-dh/documents/development.md#setup-of-bff)).
```

Comment:
nitpick: when the app is served using the `mocked` configuration, all requests are intercepted by MSW. Some might fail because there are not mocked, but no request is sent to a local BFF. If a request should be sent to the BFF, then the app must be served using the `development` configuration.

Reply:
3947b2c7c26e3fee105e42a459caf2ad3c36c50a